pkg/snapshot/snapshotter/snapshotter_test.go

```diff
@@ -117,9 +118,9 @@ var _ = Describe("Snapshotter", func() {
 	Describe("running snapshotter", func() {
 		Context("with etcd not running at configured endpoint", func() {
 			It("should timeout & not take any snapshot", func() {
-				stopCh := make(chan struct{})
-				endpoints = []string{"http://localhost:5000"}
-				etcdConnectionTimeout = 5
+				validEndpoint := etcd.Clients[0].Addr().String()
+				endpoints = []string{validEndpoint[:len(validEndpoint)-2]}
```

Comment:
This might fail if the valid endpoint itself is below 100 (it'll have only 2 digits). Maybe it's better to convert port number to int and add 1 to it. WDYT?